### DISCUSSION TITLE
Welcome to comtrya Discussions!

### DISCUSSION THREAD

**Discussion by rawkode** [comtrya/comtrya#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kraileth** [comment]
Hi all,

I'm a Unix lover, long-time Open Source advocate and blogger. As a sysadmin I have some background in configuration management and automation. While I like the ideas of Rust a lot and try to look at many of the cool things that the community does, I'm unfortunately not a developer (as in "I can't really code anything that I'd show off to others and expect them not to laugh"). My preferred platform these days is FreeBSD, and I am a very firm believer in "all open-source operating systems were created equal". IMO monopolies are always highly problematic, even in Open Source.

Besides it being a newcomer (I like it when people go ahead and start over with something - "reinventing the wheel" every decade or so is a good thing!) and written in Rust, I like the spirit of Comtrya. Building up a modern and welcoming community is an interesting and worthwhile thing. The fact that Linux distros based on the alternative Libc Musl were supported so early in the project gave me a warm feeling.

While a lot of projects today seem to target only the "major" platforms, I'm convinced that at least paying some attention to the minor ones is in fact beneficial to all platforms. Edge-cases that would probably result in bugs or even vulnerabilities later might well be caught during development instead.

So what can I do for the community? I plan on using Comtrya and providing feedback. As I like good documentation, this might also be something I could do some work on. Last but not least I already asked (via email) whether FreeBSD would be welcome as another supported platform - and as the answer was yes, I'll help with that as much as I can. Regarding this, I can:

1) provide information on how things are done on FreeBSD
2) test, test and test again
3) provide developers with SSH access to a jail upon request (I'm running a server that has some spare capacity for such experiments).

Looks like I'm also the first person to follow the invitation to GH discussions and write an introduction. Let's get the ball rolling here, too!
